schema/revert/tables/revision_status.sql

```diff
@@ -0,0 +1,14 @@
+-- Revert cif:tables/revision_status from pg
+
+begin;
+
+alter table cif.revision_status rename to amendment_status;
+alter table cif.amendment_status drop column is_amendment_specific;
+
+alter table cif.project_revision rename revision_status to amendment_status;
+
```

Comment:
You could revert the changes by doing the opposite of above 

`alter table cif.project_revision disable trigger _100_committed_changes_are_immutable, disable trigger _100_timestamps;
update cif.project_revision set revision_status='Approved' where revision_status='Applied';
alter table cif.project_revision enable trigger _100_committed_changes_are_immutable, enable trigger _100_timestamps;`

Reply:
Do we want to do this though? `Applied` and `Approved` have different meanings

Reply:
We merged them since they meant the same thing - and once they've been changed we don't have a way to revert the values anyways, this will be a one-way migration